### ISSUE TITLE
Works but always hangs for a while at 94%

### ISSUE THREAD

**Issue by yeboahnanaosei** [artemanufrij/imageburner#22]
Hi, great work bro. Image burner is now my default app for burning bootable drives.
However, every time I use it, it hangs at 94%, always. This happens every single time I've used it. No exception.
![screenshot from 2017-11-07 20-16-20](https://user-images.githubusercontent.com/22800157/32515900-709ff4cc-c3f9-11e7-8d03-fdb0178547c4.png)

There are times it tells me image burner is not responding. However after clicking wait and waiting for sometime, the job gets done.
![screenshot from 2017-11-07 20-16-14](https://user-images.githubusercontent.com/22800157/32515882-6594be82-c3f9-11e7-970c-597673785a10.png)

---
It is very confusing at times

**Comment by artemanufrij** [comment]
Yes I plan to switch to `udisk`, but I want to finish music player and video player as first. #3

**Comment by yeboahnanaosei** [comment]
Ok
